source/module_esolver/esolver_ks_lcao.cpp

```diff
@@ -55,6 +55,12 @@
 #include "module_io/write_dmr.h"
 #include "module_io/write_wfc_nao.h"
 
+// test RDMFT
+#include "module_rdmft/rdmft.h"
+#include "module_rdmft/rdmft_tools.h"
+#include "module_rdmft/rdmft_test.h"
```

Comment:
there might be a not nice dependency between functionas declared in rdmft, _tools and _test. The concept is, you only need to expose an interface to rdmft in esolver_ks_lcao, which is a derived driver. @maki49 Could you help him design a littble bit?

Reply:
Yes, that's exactly what you understood, and may need your help to complete it better